Lib/email/_parseaddr.py

```diff
@@ -394,6 +399,9 @@ def getdomain(self):
             elif self.field[self.pos] == '.':
                 self.pos += 1
                 sdlist.append('.')
+            elif self.field[self.pos] == '@':
+                #34155 Don't parse domain for a@malicious.org@important.com
```

Comment:
Can you please make this
```
# bpo-34155: Don't parse domains with two `@` like `a@malicious.org@important.com`.
```

We prefer to use `bpo-` to link to issues in the code, which stands for bugs.python.org :)

Reply:
The comment was made to fit without exceeding 80 caracters :joy:

Would this work for you ?

```
                # bpo-34155: Don't parse domains with two `@` like
                # `a@malicious.org@important.com`.
```

Or

```
# bpo-34155: Don't parse two @ like a@malicious@ok.com
```

Reply:
I think two lines are fine, we don't really need to fit it into a single line.

Reply:
Thanks for your reply it's pushed !